starter-code/public/scripts/models/repo.js

```diff
@@ -7,11 +7,23 @@ var app = app || {};
   repos.all = [];
 
   repos.requestRepos = function(callback) {
-    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
+
+    $('#about').show().siblings().hide(); // REVIEW: We have a slight refactor in selectors here, which has reduced the amount of code from the last lab.
+    $.ajax({
+      url: 'http://api.github.com/users/dustinyschild/repos',
```

Comment:
It's up to you if you use client-side authorization ([example](https://github.com/DeltaVCode-cr-301-2017/13-production-deployment/blob/7b8bca8d9d88c1f1e14e084df5125df7b549921e/public/scripts/models/repo.js#L15-L19)) or the server-side proxy, but this should use an auth token to request `/user/repos` instead of making an anonymous request.